src/config/wmodules-vuln-detector.c

```diff
@@ -611,13 +613,10 @@ int wm_vuldet_read_provider(const OS_XML *xml, xml_node *node, update_node **upd
     }
 
     /**
-    *  multi_provider = NVD, RedHat JSON and MSU.
+    *  multi_provider = NVD and MSU.
     *  Those which use <path> or <url> tags.
     **/
     if (multi_provider || (p_options.multi_path || p_options.multi_url)) {
-        // Only the JSON feed of RedHat is multi_provider
```

Comment:
Are we sure we understand the implications of skipping the multi-provider configuration steps for JREDHAT ?

Reply:
In the function `wm_vuldet_read_provider()` when ptr_name is "redhat" it will never set the variable `multi_provider`.
The function `wm_vuldet_provider_type()` sets `multi_provider` only for NVD, MSU, and ArchLinux.
In fact, the update_node for JREDHAT creation doesn't occur in `wm_vuldet_read_provider()` but in `wm_vuldet_enable_rhel_json_feed()` through the function `wm_vuldet_set_feed_version()`.

Reply:
I think this mechanism is related to the offline feeds, as we can see in https://github.com/wazuh/wazuh/pull/5352